scene/gui/tree.cpp

```diff
@@ -778,6 +778,9 @@ void TreeItem::_bind_methods() {
 
 	ClassDB::bind_method(D_METHOD("set_text", "column", "text"), &TreeItem::set_text);
 	ClassDB::bind_method(D_METHOD("get_text", "column"), &TreeItem::get_text);
+	
```

Comment:
Trailing whitespace here (even though GitHub doesn't display it for some reason - it used to).